### DISCUSSION TITLE
Trino JDBC Driver Type

### DISCUSSION THREAD

**Discussion by Xenqifi** [trinodb/trino#13358] [Q&A] [answered]
Good Day,

The client is busy setting up Trino in a dev environment.

We want to test connectivity from an Informatica perspective, however the prerequisite for the JDBC connection is that Trino must have a Type-4 compliant driver.

We had a similar issue with Presto.

Does Trino currenty provide Type 4 drivers?

Regards

**Comment by martint** [comment, marked as answer]
Yes, the Trino jdbc driver is a type 4 driver.
